runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessSearch.ts

```diff
@@ -5,6 +5,8 @@ import { GridFilterModel } from '@mui/x-data-grid';
 import { useSearchParams } from 'next/navigation';
 import { useRouter } from 'next/router';
 
+import { OrderDirection, OrderPropertyName } from 'runbotics-common';
```

Comment:
I suggest making object/enum like OrderPropertyName which will deppend on type of entity in future.